packages/jellyfish-api-core/src/category/spv.ts

```diff
@@ -1,6 +1,8 @@
 import { ApiClient } from '../.'
 import BigNumber from 'bignumber.js'
 
+export const HTLC_MINIMUM_BLOCK_COUNT = 9
+
 /**
  * SPV RPCs for DeFi Blockchain
```

Comment:
these SPV RPCs are for Bitcoin network(for now). maybe rename to `SPV RPCs for Bitcoin Blockchain`
open for discussion. @fuxingloh

Reply:
Yes, this make sense